tools/journeys/internal/journeys/cli.go

```
@@ -62,6 +66,13 @@ func Main(args []string, stdout, stderr io.Writer) int {
 	if rep.Summary.Overall == "pass" {
 		return ExitOK
 	}
+	// IR #72：模拟态失败 = 债务而非信号（ADR-0002 同一阶段化哲学），
+	// 不阻断但必须醒目可见；真实 driver 接入后 DriverMode 不再是 simulated，自然回到旧执法。
+	if rep.SimulationDebt && !*strict {
+		fmt.Fprintf(stdout, "SIMULATION-DEBT: %d 旅程失败（driver=simulated 桩噪声，不代表产品行为；真实 T20 driver 落地后自动转真实执法，--strict 可恢复阻断）\n", rep.Summary.Fail)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)